salt/metalk8s/kubeadm/init/kubeconfig/kubelet.sls

```diff
@@ -0,0 +1,7 @@
+{%- from 'metalk8s/kubeadm/init/kubeconfig/lib.sls' import kubeconfig with context %}
+
+{%- set hostname = salt.network.get_hostname() %}
```

Comment:
I don't know if we want the hostname or the fqdn for me it's more the fqdn

Reply:
In fact the default value set by kubeadm is the hostname provided by the `hostname` cmd: https://kubernetes.io/docs/reference/access-authn-authz/node/
But in fact the hostname should be a fqdn so it should be ok